core/src/main/java/com/scalar/db/api/LikeExpression.java

```diff
@@ -0,0 +1,152 @@
+package com.scalar.db.api;
+
+import com.google.common.base.MoreObjects;
+import com.scalar.db.io.TextColumn;
+import java.util.Objects;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public class LikeExpression extends ConditionalExpression {
+  private static final String DEFAULT_ESCAPE_CHAR = "\\";
+  @Nonnull private final String escape;
+  @Nonnull private final String regexPattern;
+
+  /**
+   * Constructs a {@code LikeExpression} with the specified column and operator.
+   *
+   * @param column a target column used to compare
+   * @param operator an operator used to compare the target column
+   */
+  LikeExpression(TextColumn column, Operator operator) {
+    this(column, operator, DEFAULT_ESCAPE_CHAR);
+  }
+
+  /**
+   * Constructs a {@code LikeExpression} with the specified column, operator and escape character.
+   *
+   * @param column a target column used to compare
+   * @param operator an operator used to compare the target column
+   * @param escape an escape character for the like operator
```

Comment:
Passing empty string looks to disable escapes. How about describing the behavior?

Reply:
Fixed in 9654df0.